pkg/operator/bootstrap.go

```diff
@@ -4,20 +4,21 @@ import (
 	"context"
 	"crypto/rand"
 	"fmt"
-
```

Comment:
nit: this empty line should not be removed.

Reply:
Done